src-npm/deps.cljs

```diff
@@ -0,0 +1,2 @@
+{:npm-deps {"codemirror" "^5.31.0"
+            "parinfer-codemirror" "^1.4.2"}}
```

Comment:
interesting, I thought shadow-cljs was not going to support this: https://code.thheller.com/blog/shadow-cljs/2017/09/15/js-dependencies-going-forward.html#npm-deps

Reply:
I added support a couple versions back. Still not a fan but we need a way for CLJS libs to declare dependencies on npm packages. Don't want to invent a totally new thing just "because". It works ok enough but it is still undefined how conflicts should be handled.